presto-kafka/src/test/java/io/prestosql/plugin/kafka/KafkaSecurityConfigTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka;
+
+import io.prestosql.testing.assertions.Assert;
+import org.apache.kafka.common.security.auth.SecurityProtocol;
+import org.junit.Test;
```

Comment:
```suggestion
import org.testng.annotations.Test;
```

Now we don't have to add Junit as a dependency

Reply:
done